dirsrvtests/tests/suites/entryuuid/replicated_test.py

```diff
@@ -0,0 +1,59 @@
+# --- BEGIN COPYRIGHT BLOCK ---
+# Copyright (C) 2020 William Brown <william@blackhats.net.au>
+# All rights reserved.
+#
+# License: GPL (version 3 or any later version).
+# See LICENSE for details.
+# --- END COPYRIGHT BLOCK ---
+
+import ldap
+import pytest
+import logging
+from lib389.topologies import topology_m2 as topo_m2
+from lib389.idm.user import nsUserAccounts, UserAccounts
+from lib389.paths import Paths
+from lib389.utils import ds_is_older
+from lib389._constants import *
+from lib389.replica import ReplicationManager
+
+default_paths = Paths()
+
+pytestmark = pytest.mark.tier1
+
+@pytest.mark.skipif(not default_paths.rust_enabled or ds_is_older('1.4.2.0'), reason="Entryuuid is not available in older versions")
+
+def test_entryuuid_with_replication(topo_m2):
+    """ Check that entryuuid works with replication
+
+    :id: a5f15bf9-7f63-473a-840c-b9037b787024
+
+    :setup: two node mmr
+
+    :steps:
+        1. Create an entry on one server
+        2. Wait for replication
+        3. Assert it is on the second
+
+    :expectedresults:
+        1. Success
+        1. Success
+        1. Success
+    """
+
+    server_a = topo_m2.ms["master1"]
+    server_b = topo_m2.ms["master2"]
+
+    repl = ReplicationManager(DEFAULT_SUFFIX)
+
+    account_a = nsUserAccounts(server_a, DEFAULT_SUFFIX).create_test_user(uid=2000)
+    euuid = account_a.get_attr_val_utf8('entryUUID')
+    assert(euuid is not None)
+
+    repl.wait_for_replication(server_a, server_b)
+
+    account_b = nsUserAccounts(server_b, DEFAULT_SUFFIX).get("test_user_2000")
+    euuid = account_b.get_attr_val_utf8('entryUUID')
+    assert(euuid is not None)
```

Comment:
Shouldn't this be checking for multiple entryUUID attributes?

Reply:
Yeah, I can check that too. Actually, I guess if there were multiple then any future mods of the entry would fail too?

Reply:
IDK, it allows two attributes.  Schema checks "probably" happen before backend post op plugins are called.

Reply:
The schema doesn't allow two attrs, and the plugin itself also checks if the value already exists. So I'm going to expand this test a bit today.

Reply:
Right expanding the test I have a reproducer. Looking at it now.